src/easymigration/scripts/list_bagstore_files.py

```diff
@@ -0,0 +1,88 @@
+# given a doi, retrieve the uuid and subsequently the files from the bag-store
+
+import argparse
+import csv
+import logging
+import sys
+
+import requests
+from xml.dom import minidom
+from easymigration.batch_processing import batch_process
+from easymigration.config import init
+from easymigration.pids_handling import load_pids
+
+
+def find_files(doi, dark_url, csv_writer):
+    uuid = find_uuid(doi, dark_url)
+    files_xml = get_files_xml(uuid, dark_url)
+    parse_files_xml(doi, files_xml, csv_writer)
+
+
+def find_uuid(doi, bag_index_url):
+    #locate in bag-index
+    try:
+        params = {'doi': doi}
+        response = requests.put(
+            bag_index_url + '/bag-index/search',
+            params=params)
+        response.raise_for_status()
+    except requests.exceptions.RequestException as re:
+        print("RequestException: ", re)
+        raise
+    resp_data = response.json()['result'][0]["bag-info"]["urn"]
+    return resp_data
+
+
+def get_files_xml(urn, bag_store_url):
+    try:
+        url = "{}/bag-store/bags/{}/metadata/files.xml".format(bag_store_url, urn)
+        logging.debug(url)
+        response = requests.put(url)
+        response.raise_for_status()
+    except requests.exceptions.RequestException as re:
+        print("RequestException: ", re)
+        raise
```

Comment:
just raise without an exception? Intersting discussion: https://stackoverflow.com/questions/2052390/manually-raising-throwing-an-exception-in-python